src/SignalR/server/Core/src/HubLifetimeContext.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Could we turn on nullability for new files?

Reply:
Z E S T Y. I kinda like it, it's just a bit of a pain to do. Sounds like a good topic for DoI to discuss/resolve separately from this PR though.

Reply:
I like it. I've already been [doing that](https://github.com/dotnet/aspnetcore/blob/ec0dbd0387f7fc0a13a31291d3008917910d9a1c/src/Servers/Kestrel/Core/src/Internal/Infrastructure/TransportManager.cs#L4) in some of my PRs.

Reply:
@JamesNK has been adding it to new files: https://github.com/dotnet/aspnetcore/pull/21731/files#diff-582cd4b9cdea2d10ec350424f32803f5R7. I think the runtime team did something like this.

Reply:
👍